bgp/modules/terms.py

```diff
@@ -113,3 +115,37 @@ def validate_isbn(isbn):
     def __init__(self):
         super(IsbnExtractorModule, self).__init__(self.validate_isbn)
 
+
+class PageTypeProcessor():
+
+    def __init__(self, modules):
+        """
+        :param modules: a dict of {'name': module}
+        """
+        self.modules = modules
+
+class CopyrightPageDetectorModule:
+    
+    def __init__(self, keyword):
+        self.keyword = keyword
```

Comment:
Why would the CopyrightPageDetectorModule need the parameter keyword shouldn't the keyword `Copyright`. Shouldnt the self.keyword be assigned to `copyright`?